Tests/test_SffIO.py

```diff
@@ -368,181 +368,104 @@ def test_index2(self):
 
 
 class TestSelf(unittest.TestCase):
-    """
-    These tests were originally seltests run in SffIO.py
-    """
+    """ These tests were originally seltests run in SffIO.py """
```

Comment:
The typo is still there, seltests [sic] rather than selftests, but if that's the only remaining issue I'll fix it as part of the merge...